### PR TITLE
add mobile menu

### PR DESCRIPTION
**Pull request by mixcas**
and functionality

### CODE REVIEW

File: header.php [hunk 1]
```diff
@@ -47,4 +47,42 @@
...
+            <span class="open-menu">
+              <?php echo url_get_contents(get_template_directory_uri() . '/dist/img/menu.svg'); ?>
+            </span>
+            <span class="close-menu">
+              <?php echo url_get_contents(get_template_directory_uri() . '/dist/img/cancel.svg'); ?>
+            </span>
+
+</span>
+        </div>
+        <div class="grid-item item-s-1">
```
Comment (varffs) [inline header.php]: this needs to be 10 cols wide and text aligned center

File: header.php [hunk 2]
```diff
@@ -47,4 +47,42 @@
...
+          <h1 class="u-hidden"><?php bloginfo('name'); ?></h1>
+          <div id="header-logo-holder">
+            <?php echo url_get_contents(get_template_directory_uri() . '/dist/img/formos_logo.svg'); ?>
+          </div>
+        </div>
+        <div class="grid-item item-s-1">
```
Comment (varffs) [inline header.php]: as the grid doesnt fill without content this doesnt do anything

File: src/js/main.js [hunk 3]
```diff
@@ -19,7 +19,7 @@ Site = {
     $(document).ready(function () {
       _this.SplashVideo.init();
       _this.WhatIsVideo.init();
-      _this.Modules.init();
+      //_this.Modules.init();
```
Comment (varffs) [inline src/js/main.js:22]: why?

File: src/styl/responsive/mobile.styl [hunk 4]
```diff
@@ -19,6 +29,45 @@
...
     height: 1.7rem
 
+#mobile-header
+  position: fixed
+  width: 100%
+  z-index: 1000
+
+#menu-toggle
+  svg
+    width: 50%
```
Comment (varffs) [inline src/styl/responsive/mobile.styl]: This looks much better at 100%

Comment (varffs) [reply src/styl/responsive/mobile.styl]: Even with 100% on the SVG the hit area is pretty small

**Review by varffs — CHANGES_REQUESTED** [review]

File: header.php (hunk 1 above)
Comment (mikeyrayvon) [reply header.php]: or. lets put the toggle in s-4 left aligned, and the logo in s-4 centered, and then we have more space to set the widths of each visually

File: src/styl/responsive/mobile.styl (hunk 4 above)
Comment (mikeyrayvon) [reply src/styl/responsive/mobile.styl]: relying on the width of an s-1 doesnt work. the toggle is way too big on mobile landscape

screen shot 2017-08-31 at 11 56 37 am

File: src/js/main.js (hunk 3 above)
Comment (mixcas) [reply src/js/main.js:22]: Cuz it's broken in my local due to a data structure change

**Pull request merged**